examples/e-commerce/src/app.mobile.css

```diff
@@ -0,0 +1,261 @@
+@media (max-width: 899px) {
+  /* Filters overlay */
+
+  .container-filters {
+    background: #fff;
+    border-radius: 16px;
+    left: 0;
+    max-width: initial;
+    position: fixed;
+    top: 0;
+    transform: translateY(120vh);
+    transition: transform 300ms cubic-bezier(0.465, 0.183, 0.153, 0.946);
+    width: 100%;
+    will-change: transform;
+    z-index: 1;
+  }
+
+  .container-filters .container-body,
+  .container-filters .container-filters-footer {
+    opacity: 0;
+    transition: opacity 300ms ease-in;
+    transition-delay: 200ms;
+  }
+
+  .filtering .container-filters .container-body,
+  .filtering .container-filters .container-filters-footer {
+    opacity: 1;
+    transition-delay: 0;
+  }
+
+  .container-filters .container-header,
+  .container-filters .container-body {
+    padding: 2rem 2rem 0 2rem;
+  }
+
+  .filtering .header-logo {
+    left: 50%;
+    position: absolute;
+    top: 1.5rem;
+    transform: translateX(-50%);
+  }
+
+  .filtering .header-title {
+    display: none;
+  }
+
+  .filtering .container-filters {
+    position: absolute;
+    transform: translateY(4rem);
+  }
+
+  .filtering .container-results {
+    display: none;
+  }
+
+  .container-filters-footer {
+    background-color: #fff;
+    border-top: 1px solid #ebecf3;
+    bottom: 4rem;
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem;
+    position: sticky;
```

Comment:
it looks like this didn't work on iphoneX (found on react-instantsearch example too)
![2019-06-21 17 43 12](https://user-images.githubusercontent.com/2982512/59934579-2757c680-944c-11e9-912e-8ba46fe4c257.gif)

Reply:
I guess we need `position: -webkit-sticky` to make it work. But doing so introduces another problem on my iPhone XR. 

![RPReplay_Final1561380846 2 2019-06-24 15_19_58](https://user-images.githubusercontent.com/499898/60022189-9c690d00-9693-11e9-9dd8-7ef1e518129e.gif)

Look at the bottom. The bottom bars are not aligned to bottom.

Reply:
Fixed in 1ddb6fe9343ae223bf4f4206fd42eb55eff6012d using `position: fixed;`.